README.md

```diff
@@ -49,6 +49,7 @@ xcodebuild \
 ```shell
 APP=DemoApp/build/Build/Products/Debug-iphoneos/DemoApp.app \
 TEST_APP=DemoApp/build/Build/Products/Debug-iphoneos/DemoAppUITests-Runner.app \
+TEST_LIST_FILE=DemoApp/test_classes.txt \
```

Comment:
It'd be good to explain where this file comes from. It's not immediately obvious that 1. there's a shell script as part of the build process that generates this file; 2. _what_ we are trying to demonstrate here by providing this file to saucectl; and 3. whether it's mandatory to have this at all times.